### ISSUE TITLE
Move documentation to pluginkollektiv.org

### ISSUE THREAD

**Issue by Zodiac1978** [pluginkollektiv/antispam-bee#371]
The documentation is now on:
https://antispambee.pluginkollektiv.org/documentation/
and
https://antispambee.pluginkollektiv.org/de/dokumentation/

We should add a link on each wiki page to the new page on the website, so that no link is broken in the forums and on websites linking to the wiki. Unfortunately there is o way to redirect those pages. If there is, please add a comment how to do this!

Furthermore we are looking for a way to add those link icons to headline to fast copy a link to the relevant section of the documentation. Something like: https://wordpress.org/plugins/add-anchor-links/

**Comment by Zodiac1978** [comment]
Or as an alternative, something like a TOC plugin:
https://de.wordpress.org/plugins/search/table+of+contents/

**Comment by stkjj** [comment]
can recommend https://de.wordpress.org/plugins/luckywp-table-of-contents/ as it comes as Gutenberg Block. Use this normally on almost all of my privacy policy pages to create a fast overview, including a direct access to the various headlines. E.g https://popularimages.org/de/datenschutz/

**Comment by stkjj** [comment]
Redirect: does this help? https://dev.to/steveblue/setup-a-redirect-on-github-pages-1ok7

**Comment by Zodiac1978** [comment]
> Redirect: does this help? https://dev.to/steveblue/setup-a-redirect-on-github-pages-1ok7

No, this about Github pages. Not the wiki feature :(

**Comment by pfefferle** [comment]
@Zodiac1978 but the wiki seems to be very similar with GitHub pages... I wouldn't wonder if it is the same tech behind...

See: https://github.com/pluginkollektiv/antispam-bee.wiki.git

**Comment by stkjj** [comment]
yep. That's why I suggested it. I remember the Hackathon in OS where I accidentally ended up in the git-repo for the wiki

**Comment by Zodiac1978** [comment]
Toolbelt is providing such a feature for heading anchors:
https://github.com/BinaryMoon/wp-toolbelt/wiki/Heading-Anchor

As seen here: https://pixolin.de/wieso-werden-in-wordpress-passwoerter-gehasht/

**Comment by krafit** [comment]
I've added anchor links to the documentation on pluginkollektiv.org: https://antispambee.pluginkollektiv.org/documentation/#block-comments-from-specific-countries
